admin/class-boldgrid-backup-admin-core.php

```diff
@@ -746,6 +750,45 @@ public function init_premium() {
 			return;
 		}
 
+		/*
+		 * If this version of the backup plugin is not compatible with the premium version:
+		 * 1. Don't init the premium extension.
+		 * 2. Display an admin notice telling the user to upgrade.
+		 */
+		if ( ! $this->support->is_premium_compatible() ) {
+			if ( 'update-core.php' !== $pagenow ) {
+				add_action( 'admin_notices', function() {
+					$message = wp_kses(
+						sprintf(
+							// Translators: 1 The minimum version required, 2 an opening strong tag, 3 its closing strong tag, 4 an opening anchor to the updates page, 5 its closing anchor.
+							__( 'The version of the %2$sBoldGrid Backup Premium%3$s plugin you have installed requires %2$sBoldGrid Backup%3$s version %1$s or higher. Don\'t worry, the fix is simple. Please go to your %4$sUpdates page%5$s and update the %2$sBoldGrid Backup%3$s plugin.', 'boldgrid-backup' ),
+							BOLDGRID_BACKUP_MIN_VERSION_FOR_PREMIUM,
```

Comment:
`Sep 24 10:36:25 lax-exe-dev-wpbex2 php-cgi: PHP Notice:  Use of undefined constant BOLDGRID_BACKUP_MIN_VERSION_FOR_PREMIUM - assumed 'BOLDGRID_BACKUP_MIN_VERSION_FOR_PREMIUM' in /home/joec/public_html/wp-content/plugins/boldgrid-backup/admin/class-boldgrid-backup-admin-core.php on line 765`